tmou-mapa-backend/src/main.rs

```diff
@@ -447,6 +447,112 @@ fn admin_standings(_admin: Admin, conn: PostgresDbConn) -> Result<Json<api::Stan
     }
 }
 
+////////////////
+// item editing
+////////////////
+
+#[allow(unused)]
+#[get("/admin/items")]
+fn admin_get_items(
+    _admin: Admin,
+    conn: PostgresDbConn,
+) -> Result<Json<Vec<api::ItemWithNodes>>, Status> {
+    let db_ctrl = PostgresDb::new(conn);
+    // not implemented
+    match admin::get_items(&db_ctrl) {
+        Ok(items) => Ok(Json(items)),
+        Err(_) => Err(Status::NotFound),
+    }
+}
+
+#[allow(unused)]
+#[post("/admin/items", data = "<item>")]
+fn admin_post_item(
+    _admin: Admin,
+    conn: PostgresDbConn,
+    item: Json<api::ItemCreate>,
+) -> Result<Status, Status> {
+    let db_ctrl = PostgresDb::new(conn);
+    todo!()
+}
+
+#[allow(unused)]
+#[delete("/admin/items/<item_name>")]
+fn admin_delete_item(
+    _admin: Admin,
+    conn: PostgresDbConn,
+    item_name: String,
+) -> Result<Status, Status> {
+    let db_ctrl = PostgresDb::new(conn);
+    todo!()
+}
+
+#[allow(unused)]
+#[put("/admin/items", data = "<item>")]
+fn admin_put_item(
+    _admin: Admin,
+    conn: PostgresDbConn,
+    item: Json<Vec<api::ItemUpdate>>,
+) -> Result<Status, Status> {
+    let db_ctrl = PostgresDb::new(conn);
+    todo!()
+}
+
+#[allow(unused)]
+#[post("/admin/items/<item_name>/nodes/<node_id>")]
```

Comment:
Pokud si pamatuji správně včerejší diskusi, takže tohle by měl být pouze `"/admin/items/<item_name>/nodes` a payload by měl být vector node_ids. Aby se dalo vložít více uzlů naráz a nemuselo se to dělat po jednom.

Reply:
máš pravdu, mám to v poznámkách takhle a nechal jsem se unést. A ještě mi chybí routa delete všechny nody. Tu taky doplním.